bundle/manifests/external-dns_rbac.authorization.k8s.io_v1_clusterrolebinding.yaml

```diff
@@ -11,3 +11,6 @@ subjects:
 - kind: Group
   name: system:serviceaccounts:external-dns
   namespace: external-dns
```

Comment:
serviceAccount `external-dns` still exists (used by operands) but namespace would become `external-dns-operator`, right?

Reply:
The service accounts for the operand will be created in `external-dns-operator` namespace now. If the operator is installed on a new cluster `external-dns` namespace won't be there, same about service accounts in it. If the operator is upgraded on a cluster which already had it, then yes the service accounts will still be there until the migration procedure is followed.
This binding is not strictly necessary anymore, I kept it just in case, the operator can be used in both modes with it.